lib/registries/addon/components/editable-field/styles.scss

```diff
@@ -0,0 +1,20 @@
+.Editable {
+    display: flex;
+
+    :global(.btn.btn-link) {
+        padding: 0;
+        margin-left: 15px;
+        color: $primary-blue;
+
+        :global(.fa.fa-pencil) {
+            position: relative;
+            top: 5px;
```

Comment:
instead of this, try `align-items: baseline;` on the flex parent, so the icon will be aligned with the header text.

Reply:
neat!